tests/createProjectDriver.py

```diff
@@ -0,0 +1,136 @@
+
+from loginDriver import logIn
+from selenium.webdriver import Chrome
+import time
+
+class createProject:
```

Comment:
for all classes, use camel case (e.g., CreateProject). Use the refactoring tools in your IDE to do this automatically.

Reply:
All classes in the current state of the test suite use camel case naming convention.